system/lib/libc/musl/src/thread/pthread_barrier_wait.c

```diff
@@ -88,17 +88,16 @@ int pthread_barrier_wait(pthread_barrier_t *b)
 			a_spin();
 		a_inc(&inst->finished);
 #ifdef __EMSCRIPTEN__
-		int is_runtime_thread = emscripten_is_main_runtime_thread();
+		const int is_runtime_thread = emscripten_is_main_runtime_thread();
```

Comment:
I think think this make any difference for basic types like int.

Reply:
Ah yes, reverted with commit 4de581ac1ba86a6937974736fc1d96cc9aebb447.